doc/release-notes.md

```diff
@@ -1,9 +1,9 @@
-Dogecoin Core version 1.14.2 is now available from:
```

Comment:
let's discard all changes to this file and separately do it when we have a full release scope